practices/development.md

```diff
@@ -74,6 +74,32 @@ The Sprint Board is broken down into these columns:
 
 In addition, we have a few other pieces of metadata to signal different kind of actions that would be needed 
 
+
+(coordination:team-syncs)=
+## Daily team syncs
+
+Throughout the week we have a lightweight asynchronous team sync process so that we can get on the same page.
```

Comment:
```suggestion
Throughout the week we have a lightweight asynchronous team sync process so that we can get on the same page.
These currently happen on Monday, Wednesday, and Friday in the morning time of each team member.
```